magenta-lib/src/main/scala/magenta/deployment_type/AutoScaling.scala

```diff
@@ -53,6 +57,10 @@ object AutoScaling extends DeploymentType {
     "Whether the uploaded artifacts should be given the PublicRead Canned ACL"
   ).default(false)
 
+  val asgMigrationInProgress = Param[Boolean]("asgMigrationInProgress",
```

Comment:
minor: Do we want to mention that their tags should be the same, with the new one having the `gu:riffraff:new-asg` tag? Not entirely sure where this string gets used though...